examples/StandardFirmata/StandardFirmata.ino

```diff
@@ -267,7 +275,11 @@ void setPinModeCallback(byte pin, int mode)
     case PWM:
       if (IS_PIN_PWM(pin)) {
         pinMode(PIN_TO_PWM(pin), OUTPUT);
+#if defined(MPIDE)
```

Comment:
Abstract lines 278 - 282 into a new function called servoWrite and also use it for the block beginning on line 320.